.travis/install-llvm-and-runtime-compiler.sh

```diff
@@ -7,6 +7,9 @@ if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
   sudo apt-get install -y llvm-${LLVM_VERSION}-tools clang-${LLVM_VERSION}
   sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-${LLVM_VERSION} 20
   sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-${LLVM_VERSION} 20
+
+  sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 60 --slave /usr/bin/g++ g++ /usr/bin/g++-7
```

Comment:
I am not very familiar with Debian/Ubuntu but doesn't GCC-7 become the default now? I use CC/CXX (via before_install) to test different compilers...

Reply:
Well, yes, I currently need gcc7 for valid asan builds and want to use ccache.
This is more a prototype as a WIP.